backend/tests/integration/records/api/test_api_institutions.py

```diff
@@ -171,3 +172,81 @@ def test_aut_citation_count_property_blows_up_on_wrong_pid_type(base_app, db, es
 
     with pytest.raises(AttributeError):
         record.citation_count
+
+
+def test_deleted_institution_deletes_relations_in_institution_literature_table(
```

Comment:
Hmm I think we are missing the opposite; delete literature deletes relation with institution.

Reply:
This is tested in the ```test_api_mixins``` https://github.com/inspirehep/inspirehep/pull/864/files#diff-b2fd3fb1e5f73a9192019311f091ae8aR168